services/comments.js

```diff
@@ -43,76 +31,53 @@ module.exports = class CommentsService {
     return commentModel.save();
   }
 
-  /**
-   * Adds a tag if it doesn't already exist on the comment.
-   * @throws if tag is already added to the comment
-   * @throws if tag name is not in ALLOWED_TAGS
-   * @param {String} id the id of the comment to tag
-   * @param {String} name the name of the tag to add
-   * @param {String} assigned_by the user id for the user who added the tag
-   */
-  static addTag(id, name, assigned_by) {
-
-    // Disabling allowed tags until we are able to extend them
-    // if (ALLOWED_TAGS.find((t) => t.name === name) == null) {
-    //   return Promise.reject(new Error('tag not allowed'));
-    // }
+  static addTag(id, tagLink, verifyOwnership = false) {
```

Comment:
This function and `removeTag` live in `services/tags.js` and be imported and appended to all services for 'taggable" models.